d3a_interface/constants_limits.py

```diff
@@ -226,3 +226,7 @@ class GlobalConfig:
 DATE_TIME_FORMAT = f"{DATE_FORMAT}T{TIME_FORMAT}"
 DATE_TIME_FORMAT_SECONDS = f"{DATE_FORMAT}T{TIME_FORMAT_SECONDS}"
 DATE_TIME_UI_FORMAT = "MMMM DD YYYY, HH:mm [h]"
+
+HEARTBEAT_CHANNEL = "d3a-heartbeat"
+PULSE_RATE = 5  # in secs
+TOLERANCE = 1
```

Comment:
Because tolerance is a very broad term, I would rename this to HEARTBEAT_TIMEOUT or HEARTBEAT_TOLERANCE